README.md

```diff
@@ -1,2 +1,98 @@
 # lsmlite-rs
-Rust bindings for SQLite’s lsm1 extension in stand-alone manner.
+
```

Comment:
You might want to add some badges after publishing to crates.io and docs.rs:
```
[![Cargo](https://img.shields.io/crates/v/lsmlite-rs.svg)](https://crates.io/crates/lsmlite-rs)
[![Documentation](https://docs.rs/lsmlite-rs/badge.svg)](https://docs.rs/lsmlite-rs)
```

Reply:
Added. Thanks for letting me know.